### ISSUE TITLE
error: services.pygeoapi.depends_on contains an invalid type, it should be an array

### ISSUE THREAD

**Issue by petersilva** [wmo-im/wis2node#63]
I worked on this with @webb-ben a bit... thought I would formalize it a little. When running on ubuntu 20.04 using OS provided versions of docker-compose, I get the error:

```

fractal% python3 wis2node-ctl.py update
WARNING: The WIS2NODE_PIP3_EXTRA_PACKAGES variable is not set. Defaulting to a blank string.
WARNING: The WIS2NODE_DATADIR_RETENTION_DAYS variable is not set. Defaulting to a blank string.
ERROR: The Compose file './docker/docker-compose.yml' is invalid because:
services.pygeoapi.depends_on contains an invalid type, it should be an array
fractal% 

```

There are two methods I can work around it:
1. remove the "service_healthy" depends on stanza in pygeoapi by commenting it out.
2. change the version at the head of the docker/docker-compose.yml and docker/docker-compose.override.yml
    Downgrading from "3" to "2.2"

in both cases all operations seem to work fine. but in option 1 we lose the health check.  Is there an important reason for putting "3" as the docker compose configuration version?  I don't think it is using anything specific to that version.  

I could submit a patch with either approach... you there could be another approach to recommend.

**Comment by webb-ben** [comment]
Relevant for: https://github.com/wmo-im/wis2node/tree/healthcheck

**Comment by maaikelimper** [comment]
I can't think of an important reason for using v3. 
Docker disabled the option of having a condition in depends_on because they recommend you to "to perform this check in your application code" as per https://docs.docker.com/compose/startup-order/

**Comment by tomkralidis** [comment]
Fixed via #66
